manifests/server/database.pp

```diff
@@ -2,14 +2,13 @@
 define postgresql::server::database(
   $comment          = undef,
   $dbname           = $title,
-  $owner            = $postgresql::server::user,
+  $owner            = undef,
   $tablespace       = undef,
   $template         = 'template0',
   $encoding         = $postgresql::server::encoding,
   $locale           = $postgresql::server::locale,
   $istemplate       = false,
   $connect_settings = $postgresql::server::default_connect_settings,
-  $change_ownership = false,
```

Comment:
I don't think this argument has been released yet.  Do you think it is still necessary to accept it?